dockerfiles/dev.Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM python:2.7
+
+ENV APP_HOME /srv/www/django/microweb/
+
+
+WORKDIR ${APP_HOME}
+
+RUN apt-get -qq update && \
+    apt-get -yq install --no-install-recommends \
+        build-essential \
+        fabric \
+        libevent-dev \
+        libmemcached-dev \
+        zlib1g-dev && \
+    apt-get -yq --purge autoremove && \
+    apt-get -q clean && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY requirements.txt /${APP_HOME}
+
+RUN curl https://bootstrap.pypa.io/pip/2.7/get-pip.py --output get-pip.py \
+    && python get-pip.py \
+    && rm get-pip.py \
+    && pip install virtualenv \
```

Comment:
nb. seems like you've installed `virtualenv` but not created & activated one, so the requirements will install globally? I think this is fine in a docker container, but maybe something to tidy up later?

Reply:
I'm sure you're right. I don't have much experience with Python, so if you think I'm doing something wrong, it's almost certainly because I am! I'll merge this one, but please feel free to make any changes you think it needs